src/wmnds/components/disruption-indicator/small/_small.njk

```diff
@@ -2,10 +2,10 @@
 {% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
 
 {% set iconRight = params.iconRight if params.iconRight else 'general-warning-circle' %} {# set icon right classes #}
-{% set class = ' ' + params.classModifiers if params.classModifiers %}
+{% set _class = ' ' + params.classes if params.classes %}
```

Comment:
It's not a major thing but I would name it classes instead of _class, as it is not an internal variable.. but as I said this is a minor thing...